### PR TITLE
ci: add helm chart to repo with release workflow

### PR DESCRIPTION
**Pull request by mzeevi**
This PR introduces a Helm Chart for deploying Capp, and in addition introduces a Helmfile which installs all the Capp prerequisites (with the exception of Knative) as Helm charts.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -226,24 +176,32 @@ install-knative: ## Install knative controller on the kind cluster
...
-	kubectl delete providers dana-team-provider-dns
+
+.PHONY: install-prereq-helmfile
+install-prereq-helmfile: helm helmfile
+	${HELMFILE} apply -f $(PREREQ_HELMFILE) \
+	--state-values-set providerDNSRealmName=${PROVIDER_DNS_REALM} \
+	--state-values-set providerDNSKDCName=${PROVIDER_DNS_KDC} \
+	--state-values-set providerDNSPolicy=${PROVIDER_DNS_POLICY} \
+	--state-values-set providerDNSNameservers=${PROVIDER_DNS_NAMESERVER} \
+	--state-values-set providerDNSUsername=${cccc} \
```
Comment (fran2810) [inline Makefile:195]: what is this ?

**Review by fran2810 — COMMENTED** [review]
weird  variable in make file

File: Makefile (hunk 1 above)
Comment (mzeevi) [reply Makefile]: Strange mistake, fixed

**Review by fran2810 — APPROVED** [review]
/lgtm

**Pull request merged**